Library/Homebrew/cmd/list.rb

```diff
@@ -1,3 +1,24 @@
+#:  * `ls`, `list` [`--full-name`]:
```

Comment:
I think it would be a good idea to make the canonical name (`list`) the first and list the aliases (`ls`) afterwards. Otherwise the order in the generated man page will be weird sometimes.